### PR TITLE
new config option 'statistics_interval' (defaults to 60s)

### PR DESCRIPTION
**Pull request by klaus3000**
Defines how often DSC writes statistics. Patch by michael.braunoeder@nic.at.

### CODE REVIEW

**Comment by klaus3000** [comment]
Replaces https://github.com/DNS-OARC/dsc/pull/30

File: src/config_hooks.c [hunk 1]
```diff
@@ -128,6 +129,13 @@ dataset_cmpfunc(const void *a, const void *b)
     return strcasecmp(a, b);
 }
 int
+set_statistics_interval (const char *s)
+{
+    syslog(LOG_INFO, "Setting statistics interval to: %s", s);
+    statistics_interval = strtoull(s, NULL, 10);
```
Comment (jelu) [inline src/config_hooks.c]: Don't allow interval to be zero.

**Comment by jelu** [comment]
#build

**Comment by jelu** [comment]
#build

**Comment by jelu** [comment]
#build

**Comment by jelu** [comment]
#build

**Comment by jelu** [comment]
#build

**Comment by jelu** [comment]
#build

**Comment by jelu** [comment]
Failure is because of #62

**Comment by jelu** [comment]
#build

**Comment by jelu** [comment]
Replaced by #64

**Pull request closed without merging**